data/bedrock/latest/proto.yml

```diff
@@ -3964,15 +3964,7 @@ packet_client_cheat_ability:
 packet_camera_presets:
    !id: 0xc6
    !bound: client
-   # Name is the name of the preset. Each preset must have their own unique name.
-   name: string
-   # Parent is the name of the preset that this preset extends upon. This can be left empty.
-   parent: string
-   position: Vec3fopts
-   rotation: Vec2fopts
-   ## TODO: make this an enum afer adding proper optional support inside pdefyaml
-   audio_listener?: u8
-   player_effects?: bool
+   presets: CameraPresets[]u8
```

Comment:
Actually, this is not a u8. It should be a varint.

Reply:
Fixed.